File: persephone/provisioning-service/src/main/java/com/vmware/blockchain/deployment/services/orchestration/model/vsphere/DatastoreSummary.java
/*
 * Copyright (c) 2020 VMware, Inc. All rights reserved. VMware Confidential
 */

package com.vmware.blockchain.deployment.services.orchestration.model.vsphere;

import lombok.Data;

/**
 * Data class.
 */
@Data
public class DatastoreSummary {
    String datastore;
    String name;
    String type;
    Long freeSpace;
    Long capacity;
}

File: persephone/model/src/common/com/vmware/blockchain/model/sddc/DatastoreSummary.kt
/* **********************************************************************
 * Copyright 2018 VMware, Inc.  All rights reserved. VMware Confidential
 * *********************************************************************/
package com.vmware.blockchain.model.sddc

data class DatastoreSummary(
    val datastore: String,
    val name: String,
    val type: String,
    val free_space: Long,
    val capacity: Long
)
